node-notes-cli/app.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
+
+let dataParse;
+let dataStringify;
+
+fs.readFile('./data.json', 'utf8', (err, data) => {
+  if (err) {
+    console.error(err);
+    process.exit(1);
+  }
+  dataParse = JSON.parse(data);
+
+  if (process.argv[2] === 'read') { // reading my data as an object
+    const notes = dataParse.notes;
+    for (const prop in notes) {
+      console.log(`${prop}: ${notes[prop]}`);
+    }
+    process.exit(1);
```

Comment:
It's better to `return` instead of `process.exit()` if everything went ok. `process.exit(1)` specifically means that the code failed.